src/mdbExtensionController.ts

```diff
@@ -48,28 +48,20 @@ export default class MDBExtensionController implements vscode.Disposable {
   _languageServerController: LanguageServerController;
   _webviewController: WebviewController;
 
-  constructor(
-    context: vscode.ExtensionContext,
-    connectionController?: ConnectionController
```

Comment:
We haven't used it anywhere, right?

Reply:
Isn't this useful to pass a mock for tests?

Reply:
I replaced ConnectionController with needTelemetry flag because I think ConnectionController is not used here at all. Just wanted to double-check that ConnectionController isn't placed here for future needs.